### PR TITLE
Holium's initial round of bounties

### PR DESCRIPTION
**Pull request by drunkplato**

### CODE REVIEW

**Comment by drunkplato** [comment]
Tome is now removed, should be good to merge.

File: content/grants/app-engram.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+grant_category = [ "App Dev" ]
+
+[extra]
+image = ""
+description = "Engram is a knowledge management application that allows individuals or teams to build rich note documents."
+reward = "$70,000 in WSTR"
+assignee = ""
+id = ""
+completed = false
+work_request_link = ""
```
Comment (jalehman) [inline content/grants/app-engram.md]: @mikeosborne can you add a work request link here please? Then this is good to go.

**Review by jalehman — APPROVED** [review]

File: content/grants/app-engram.md (hunk 1 above)
Comment (drunkplato) [reply content/grants/app-engram.md]: Honestly not sure what this is for. Used the template from the Books app grant. Is this field needed? Can remove.

Comment (jalehman) [reply content/grants/app-engram.md]: It is needed, as it's what allows people to apply for the bounty. Mike needs to add it though.

Comment (mikeosborne) [inline content/grants/app-engram.md]:
```suggestion
work_request_link = "
https://airtable.com/shr4qt9t9kz7RaOIa?prefill_Grant+ID=B0155&prefill_Grant+Name=Engram"
```

File: content/grants/app-engram.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  - video
+  - image
+
+## Technical requirements
+
+The most complex part of this is the revision log. It should be easy to revert to a certain point in time. There will also need to be a sentence parser that stores each sentence in a store with associated comments and other metadata. Each sentence or collection of sentences should be able to be referenced by other documents.
+
+Holium will work with the developer(s) to provide frontend help.
+
+## Milestone 1 - Single user documents - 30,000 WSTR
```
Comment (mikeosborne) [inline content/grants/app-engram.md]:
```suggestion
## Milestone 1 - Single user documents - $30,000 in WSTR
```

File: content/grants/app-engram.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+The first milestone is to release a version of the app that allows for single user documents, meaning there is a document owner and only they can edit it. The app should include the rich document editor features without collaboration.
+
+The app should be released.
+
+## Milestone 2 - Revision history, comments, and referencing - 15,000 WSTR
```
Comment (mikeosborne) [inline content/grants/app-engram.md]: ## Milestone 2 - Revision history, comments, and referencing - $15,000 in WSTR

File: content/grants/app-engram.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+This second milestone is building out the revision history log to allow for undo, redo, and time traveling through revision snapshots. This will serve as the foundation of the third milestone.
+
+There should be a revision history log in the UI that allows users to go back to previous snapshots as well as the ability to leave comments on the document or on sentences.
+
+## Milestone 3 - Collaboration - 25,000 WSTR
```
Comment (mikeosborne) [inline content/grants/app-engram.md]: ## Milestone 3 - Collaboration - $25,000 in WSTR

File: content/grants/app-engram.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+- At least three years of professional programming experience
+- Experience with full-stack development, including JS and some server-side language
+- Demonstrable experience with writing Gall agents
+- Coordinate with Holium on designs, frontend, and agents
+
+## Schedule
+This project has many pieces and would likely require a team size of two or more. There would need to be a significant amount of frontend work, which Holium will help build or facilitate using our design system. V1 deadline is 6 months from start.
+
+## Reward
+$70,000 WSTR (team of two)
```
Comment (mikeosborne) [inline content/grants/app-engram.md]: $70,000 in WSTR (team of two)

File: content/grants/app-engram.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+[taxonomies]
+grant_type = [ "Bounty" ]
+grant_category = [ "App Dev" ]
+
+[extra]
+image = ""
+description = "Engram is a knowledge management application that allows individuals or teams to build rich note documents."
+reward = "$70,000 in WSTR"
+assignee = ""
+id = ""
```
Comment (mikeosborne) [inline content/grants/app-engram.md]: id = "B0155"
champion = "Holium"

**Review by mikeosborne — CHANGES_REQUESTED** [review]

File: content/grants/app-engram.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+
+[extra]
+image = ""
+description = "Engram is a knowledge management application that allows individuals or teams to build rich note documents."
+reward = "$70,000 in WSTR"
+assignee = ""
+id = "B0155"
+champion = "Holium"
+completed = false
+work_request_link = "
```
Comment (mikeosborne) [inline content/grants/app-engram.md]:
```suggestion
work_request_link = "https://airtable.com/shr4qt9t9kz7RaOIa?prefill_Grant+ID=B0155&prefill_Grant+Name=Engram"
```

File: content/grants/app-engram.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+https://airtable.com/shr4qt9t9kz7RaOIa?prefill_Grant+ID=B0155&prefill_Grant+Name=Engram"
```
Comment (mikeosborne) [inline content/grants/app-engram.md]:
```suggestion

```

**Review by mikeosborne — CHANGES_REQUESTED** [review]

File: content/grants/app-engram.md (hunk 8 above)
Comment (mikeosborne) [reply content/grants/app-engram.md]: You were a little too literal with my comment. :)

Comment (drunkplato) [reply content/grants/app-engram.md]: What else should I do?

Comment (mikeosborne) [reply content/grants/app-engram.md]: make sure it is all one line with the link between the quotes

Comment (drunkplato) [reply content/grants/app-engram.md]: fixed

**Pull request merged**